domain/domain.py

```diff
@@ -332,52 +330,62 @@ def get_domain_cell(self, attr, row):
         :return: (initial value of entity-attribute, domain values for entity-attribute).
         """
 
-        domain = set([])
+        domain = set()
         correlated_attributes = self.get_corr_attributes(attr, self.cor_strength)
         # Iterate through all attributes correlated at least self.cor_strength ('cond_attr')
         # and take the top K co-occurrence values for 'attr' with the current
         # row's 'cond_attr' value.
         for cond_attr in correlated_attributes:
+            # Ignore correlations with index, tuple id or the same attribute.
             if cond_attr == attr or cond_attr == 'index' or cond_attr == '_tid_':
```

Comment:
Do we know why `index` might be included in the list of correlated attributes? We might run into issues here if there is genuinely a column in the data called `index`.